### ISSUE TITLE
Remove user as admin of RSO

### ISSUE THREAD

**Issue by imlasky** [thrakish/cop4710_project#2]
Once selected as an admin, unable to deselect

**Comment by travisbashor** [comment]
I'm at work now, but maybe this is due to some privilege issue. Only admins can remove admins once they're in the db as such? If so, maybe we could hold out on saving as admin until after the RSO is created. I'll look when I get back today.

**Comment by jstraney** [comment]
@imlasky Yeah, the javascript I wrote doesn't unset the user selected as admin (to do this, I'll have to have the current user id somewhere in the partial, and reset the hidden field with Javascript)

@travisbashor I'll have to take a second look to see what's happening. Once I get the event search working perfectly, I think we'll actually be good to go and the rest is improvement (Oh except categories for Events. I do not have this...)
